frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardInfoSidebar.module.css

```diff
@@ -0,0 +1,5 @@
+.EditableTextContainer {
+  max-height: 19rem;
+  overflow: auto;
+  line-height: 1.38rem; /* magic number to keep line-height from changing in edit mode */
```

Comment:
```suggestion
  line-height: 1.38rem; /* magic number to keep line-height from changing in edit mode */
  position: relative;
  inset-inline-start: -4px;
```

This aligns the description with the heading.

Here it is, not quite aligned:

![image](https://github.com/user-attachments/assets/d80e41b4-ce5e-4954-abbc-315e3c25c13b)

Aligned:

![image](https://github.com/user-attachments/assets/1117d6d8-6393-4f60-aa23-ebf58431ef54)

Reply:
nano-nit: in my browser `line-height: 1.37rem` reduces the jump a tiny bit better than 1.38rem!